### ISSUE TITLE
Retirement of Microsoft Forms Pro, is it still valid?

### ISSUE THREAD

**Issue by TobiasAT** [MicrosoftDocs/forms-pro#37]
Hello, is it still valid that Microsoft Forms Pro will be retired on 31. August? It's mentioned at https://docs.microsoft.com/en-us/dynamics365/customer-voice/forms-pro-transition#experience-for-users-based-on-their-forms-pro-license. In a normal case Microsoft informs customer in the M365 admin center via a message center post about retirements. I found no post from the last months. I wonder if someone has forgotten to inform customers, or the August date from the post is not valid? 

Thanks for a check.  
Tobias

**Comment by sbmjais** [comment]
Hi @TobiasAT - Thanks for contacting us. This is not a product retirement or deprecation, but product rebranding with new user interface and experience. We have also informed the tenant administrator through Message Center communication in July.

**Comment by TobiasAT** [comment]
Hello @sbmjais, thanks for the feedback. Ok, maybe it's a different perspective. You change the product name, the landing page, the prices. For me it's a retirement of Forms Pro. But yes, I saw the data were migrated to Customer Voice. I cannot confirm the tenant information. I went through all messages over the last 90 days, nothing about Forms Pro or Customer Voice. The latest message what I found was the information Forms is coming to Office Mobile app. As all of our Forms Pro users were licensed with a Customer Voice Trial license I assume Microsoft identified our tenant as a "Forms Pro tenant".
